src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseResultJenkinsIntegrationTest.java

```diff
@@ -38,20 +39,24 @@ void tearDown() {
     @Test
     @WithMockUser(value = "student1", roles = "USER")
     public void shouldUpdateTestCasesAndResultScoreFromSolutionParticipationResult() {
+        programmingExerciseResultTestService.setup();
         var notification = ModelFactory.generateTestResultDTO(Constants.ASSIGNMENT_REPO_NAME, List.of("test1", "test2", "test4"), List.of());
         programmingExerciseResultTestService.shouldUpdateTestCasesAndResultScoreFromSolutionParticipationResult(notification);
     }
 
-    @Test
+    @ParameterizedTest
+    @EnumSource(value = ProgrammingLanguage.class, names = { "JAVA" })
     @WithMockUser(value = "student1", roles = "USER")
-    public void shouldStoreFeedbackForResultWithStaticCodeAnalysisReport() {
+    public void shouldStoreFeedbackForResultWithStaticCodeAnalysisReport(ProgrammingLanguage programmingLanguage) {
+        programmingExerciseResultTestService.setupForProgrammingLanguage(programmingLanguage);
         var notification = ModelFactory.generateTestResultDTO(Constants.ASSIGNMENT_REPO_NAME, List.of("test1"), List.of());
-        programmingExerciseResultTestService.shouldStoreFeedbackForResultWithStaticCodeAnalysisReport(notification);
+        programmingExerciseResultTestService.shouldStoreFeedbackForResultWithStaticCodeAnalysisReport(notification, ProgrammingLanguage.JAVA);
```

Comment:
```suggestion
        programmingExerciseResultTestService.shouldStoreFeedbackForResultWithStaticCodeAnalysisReport(notification, programmingLanguage);
```

Reply:
resolved in bb131224aa3583ecda43327127046d7f590db25b e7d6a4a26d64212388c73f080885f6d0cf7345f3